<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Tool</class>
 <widget class="QWidget" name="Tool">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QWidget" name="widget_tool" native="true">
     <layout class="QGridLayout" name="gridLayout_3">
      <property name="leftMargin">
       <number>70</number>
      </property>
      <property name="topMargin">
       <number>50</number>
      </property>
      <property name="rightMargin">
       <number>50</number>
      </property>
      <property name="bottomMargin">
       <number>50</number>
      </property>
      <property name="horizontalSpacing">
       <number>70</number>
      </property>
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,1">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_7">
          <property name="spacing">
           <number>0</number>
          </property>
          <item>
           <widget class="QWidget" name="widget_function" native="true">
            <layout class="QGridLayout" name="gridLayout_2">
             <property name="horizontalSpacing">
              <number>5</number>
             </property>
             <item row="0" column="0">
              <widget class="QWidget" name="widget_wifi" native="true">
               <layout class="QVBoxLayout" name="verticalLayout_2">
                <property name="leftMargin">
                 <number>0</number>
                </property>
                <property name="topMargin">
                 <number>0</number>
                </property>
                <property name="rightMargin">
                 <number>0</number>
                </property>
                <property name="bottomMargin">
                 <number>0</number>
                </property>
                <item>
                 <widget class="QPushButton" name="pushButton_wifi">
                  <property name="minimumSize">
                   <size>
                    <width>120</width>
                    <height>100</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>167777</width>
                    <height>16777215</height>
                   </size>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">border-image: url(:/car_terminall/wifi.png);</string>
                  </property>
                  <property name="text">
                   <string/>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="label_wifi">
                  <property name="maximumSize">
                   <size>
                    <width>16777215</width>
                    <height>25</height>
                   </size>
                  </property>
                  <property name="font">
                   <font>
                    <pointsize>13</pointsize>
                   </font>
                  </property>
                  <property name="text">
                   <string>WiFi</string>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignCenter</set>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QWidget" name="widget_light" native="true">
               <layout class="QVBoxLayout" name="verticalLayout">
                <property name="leftMargin">
                 <number>0</number>
                </property>
                <property name="topMargin">
                 <number>0</number>
                </property>
                <property name="rightMargin">
                 <number>0</number>
                </property>
                <property name="bottomMargin">
                 <number>0</number>
                </property>
                <item>
                 <widget class="QPushButton" name="pushButton_light">
                  <property name="minimumSize">
                   <size>
                    <width>120</width>
                    <height>100</height>
                   </size>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">border-image: url(:/car_terminall/light0.png);</string>
                  </property>
                  <property name="text">
                   <string/>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="label_light">
                  <property name="maximumSize">
                   <size>
                    <width>16777215</width>
                    <height>25</height>
                   </size>
                  </property>
                  <property name="font">
                   <font>
                    <pointsize>13</pointsize>
                   </font>
                  </property>
                  <property name="text">
                   <string>灯</string>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignCenter</set>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item row="0" column="2">
              <widget class="QWidget" name="widget_backcar" native="true">
               <layout class="QVBoxLayout" name="verticalLayout_3">
                <property name="leftMargin">
                 <number>0</number>
                </property>
                <property name="topMargin">
                 <number>0</number>
                </property>
                <property name="rightMargin">
                 <number>0</number>
                </property>
                <property name="bottomMargin">
                 <number>0</number>
                </property>
                <item>
                 <widget class="QPushButton" name="pushButton_backcar">
                  <property name="minimumSize">
                   <size>
                    <width>120</width>
                    <height>100</height>
                   </size>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">border-image: url(:/car_terminall/backcar.png);</string>
                  </property>
                  <property name="text">
                   <string/>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="label_backcar">
                  <property name="maximumSize">
                   <size>
                    <width>16777215</width>
                    <height>25</height>
                   </size>
                  </property>
                  <property name="font">
                   <font>
                    <pointsize>13</pointsize>
                   </font>
                  </property>
                  <property name="text">
                   <string>倒车影像</string>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignCenter</set>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QWidget" name="widget_folder" native="true">
               <layout class="QVBoxLayout" name="verticalLayout_4">
                <property name="leftMargin">
                 <number>0</number>
                </property>
                <property name="topMargin">
                 <number>0</number>
                </property>
                <property name="rightMargin">
                 <number>0</number>
                </property>
                <property name="bottomMargin">
                 <number>0</number>
                </property>
                <item>
                 <widget class="QPushButton" name="pushButton_folder">
                  <property name="minimumSize">
                   <size>
                    <width>120</width>
                    <height>100</height>
                   </size>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">border-image: url(:/car_terminall/folder.png);</string>
                  </property>
                  <property name="text">
                   <string/>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="label_folder">
                  <property name="maximumSize">
                   <size>
                    <width>16777215</width>
                    <height>25</height>
                   </size>
                  </property>
                  <property name="font">
                   <font>
                    <pointsize>13</pointsize>
                   </font>
                  </property>
                  <property name="text">
                   <string>文件夹</string>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignCenter</set>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QWidget" name="widget_set" native="true">
               <layout class="QVBoxLayout" name="verticalLayout_5">
                <property name="leftMargin">
                 <number>0</number>
                </property>
                <property name="topMargin">
                 <number>0</number>
                </property>
                <property name="rightMargin">
                 <number>0</number>
                </property>
                <property name="bottomMargin">
                 <number>0</number>
                </property>
                <item>
                 <widget class="QPushButton" name="pushButton_set">
                  <property name="minimumSize">
                   <size>
                    <width>120</width>
                    <height>100</height>
                   </size>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">border-image: url(:/car_terminall/set.png);</string>
                  </property>
                  <property name="text">
                   <string/>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="label_set">
                  <property name="maximumSize">
                   <size>
                    <width>16777215</width>
                    <height>25</height>
                   </size>
                  </property>
                  <property name="font">
                   <font>
                    <pointsize>13</pointsize>
                   </font>
                  </property>
                  <property name="text">
                   <string>设置</string>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignCenter</set>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item row="1" column="2">
              <widget class="QWidget" name="widget_instructions" native="true">
               <layout class="QVBoxLayout" name="verticalLayout_6">
                <property name="leftMargin">
                 <number>0</number>
                </property>
                <property name="topMargin">
                 <number>0</number>
                </property>
                <property name="rightMargin">
                 <number>0</number>
                </property>
                <property name="bottomMargin">
                 <number>0</number>
                </property>
                <item>
                 <widget class="QPushButton" name="pushButton_instructions">
                  <property name="minimumSize">
                   <size>
                    <width>120</width>
                    <height>100</height>
                   </size>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">border-image: url(:/car_terminall/instructions.png);</string>
                  </property>
                  <property name="text">
                   <string/>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="label_instructions">
                  <property name="maximumSize">
                   <size>
                    <width>16777215</width>
                    <height>25</height>
                   </size>
                  </property>
                  <property name="font">
                   <font>
                    <pointsize>13</pointsize>
                   </font>
                  </property>
                  <property name="text">
                   <string>使用介绍</string>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignCenter</set>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout" stretch="5,4">
            <item>
             <widget class="QLabel" name="label_temp_pixmal">
              <property name="minimumSize">
               <size>
                <width>40</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>80</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">border-image: url(:/car_terminall/temperature.png);</string>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_temp_number">
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>80</height>
               </size>
              </property>
              <property name="font">
               <font>
                <pointsize>30</pointsize>
               </font>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QWidget" name="widget_compass" native="true">
          <property name="maximumSize">
           <size>
            <width>455</width>
            <height>16777215</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
